compiler/damlc/lib/DA/Cli/Visual.hs

```diff
@@ -74,8 +74,13 @@ startFromExpr seen world e = case e of
 startFromChoice :: LF.World -> LF.TemplateChoice -> Set.Set Action
 startFromChoice world chc = startFromExpr Set.empty world (LF.chcUpdate chc)
 
+-- We adding template name to archive as we need to have unique choice names
+archiveChoiceAndAction :: LF.Template -> ChoiceAndAction
+archiveChoiceAndAction tpl = ChoiceAndAction (LF.ChoiceName $ tplName tpl <> "_Archive") True Set.empty
+
 templatePossibleUpdates :: LF.World -> LF.Template -> [ChoiceAndAction]
-templatePossibleUpdates world tpl = map (\c -> ChoiceAndAction tpl c (startFromChoice world c)) (NM.toList (LF.tplChoices tpl))
+templatePossibleUpdates world tpl = actions ++ [archiveChoiceAndAction tpl]
```

Comment:
Will we not end up with the `Archive` choice being represented twice, once in `actions` and once in `archiveChoiceAndAction`?.